projects/packages/my-jetpack/src/class-products.php

```diff
@@ -36,6 +36,7 @@ public static function get_products_classes() {
 			'protect'    => Products\Protect::class,
 			'videopress' => Products\Videopress::class,
 			'stats'      => Products\Stats::class,
+			'ai'         => Products\Jetpack_Ai::class,
```

Comment:
Hey @CGastrell 👋,
I know it was quite a while ago when you worked on this, but do you know or remember if it was intentional to add the AI product class twice here?

As you can see, there is already a definition:
`'jetpack-ai' => Products\Jetpack_Ai::class,`

But then in this PR it was added again but under a different array key:
`'ai' => Products\Jetpack_Ai::class,`

I don't think both these entries are necessary and I believe I can remove one of them, but I just wanted to double check with you in case there is something I'm missing, or some intentional reason to have them both that is not clearly apparent.
Thanks! 😃

Reply:
I can't remember right now, but there was some mixup with how cards/tables were automatically instanced given a feature/product/module/purchase/plan, and at some point I couldn't just change it (I think it was coming from backend like that, "ai"), so the quick solution was to just add a new entry on the products_classes mapping.

If you're sure it's redundant, I'd say let's check it against MyJetpack and JP Dashboard, both should remain functional as well as any checkout process.